ScrollIntoView error TS2559: Type 'true' has no properties in common with type 'ScrollIntoViewOptions'

I am use webdriwer
I have updated the codeceptjs version (3.5.10). Now I'm getting an error on the `I.scrollIntoView('#submit', true);` method.
```
⨯ Unable to compile TypeScript:
fragments/mobile/footer/Footer.ts(32,52): error TS2559: Type 'true' has no properties in common with type 'ScrollIntoViewOptions'.
```

--- a/docs/helpers/Appium.md
+++ b/docs/helpers/Appium.md
@@ -1171,7 +1171,7 @@ I.scrollIntoView('#submit', { behavior: "smooth", block: "center", inline: "cent
 #### Parameters
 
 -   `locator` **([string][5] \| [object][11])** located by CSS|XPath|strict locator.
--   `scrollIntoViewOptions` **ScrollIntoViewOptions** see [https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView][17].
+-   `scrollIntoViewOptions` **(ScrollIntoViewOptions | [boolean][7])** either alignToTop=true|false or scrollIntoViewOptions. See [https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView][17].
 
 Returns **void** automatically synchronized promise through #recorderSupported only for web testing
 

--- a/docs/helpers/WebDriver.md
+++ b/docs/helpers/WebDriver.md
@@ -1639,7 +1639,7 @@ I.scrollIntoView('#submit', { behavior: "smooth", block: "center", inline: "cent
 #### Parameters
 
 -   `locator` **([string][17] | [object][16])** located by CSS|XPath|strict locator.
--   `scrollIntoViewOptions` **ScrollIntoViewOptions** see [https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView][33].
+-   `scrollIntoViewOptions` **(ScrollIntoViewOptions | [boolean][32])** either alignToTop=true|false or scrollIntoViewOptions. See [https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView][33].
 
 Returns **void** automatically synchronized promise through #recorder
 

--- a/docs/webapi/scrollIntoView.mustache
+++ b/docs/webapi/scrollIntoView.mustache
@@ -7,5 +7,5 @@ I.scrollIntoView('#submit', { behavior: "smooth", block: "center", inline: "cent
 ```
 
 @param {LocatorOrString} locator located by CSS|XPath|strict locator.
-@param {ScrollIntoViewOptions} scrollIntoViewOptions see https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView.
+@param {ScrollIntoViewOptions|boolean} scrollIntoViewOptions either alignToTop=true|false or scrollIntoViewOptions. See https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView.
 @returns {void} automatically synchronized promise through #recorder

--- a/typings/tests/helpers/Appium.types.ts
+++ b/typings/tests/helpers/Appium.types.ts
@@ -84,6 +84,7 @@ appium.grabValueFromAll(str); // $ExpectType Promise<string[]>
 appium.grabValueFrom(str); // $ExpectType Promise<string>
 appium.saveScreenshot(str); // $ExpectType Promise<void>
 appium.scrollIntoView(str, {}); // $ExpectType void
+appium.scrollIntoView(str, true); // $ExpectType void
 appium.seeCheckboxIsChecked(str); // $ExpectType void
 appium.seeElement(str); // $ExpectType void
 appium.seeInField(str, str); // $ExpectType void

--- a/typings/tests/helpers/AppiumTs.types.ts
+++ b/typings/tests/helpers/AppiumTs.types.ts
@@ -84,6 +84,7 @@ appium.grabValueFromAll(str); // $ExpectType Promise<string[]>
 appium.grabValueFrom(str); // $ExpectType Promise<string>
 appium.saveScreenshot(str); // $ExpectType Promise<void>
 appium.scrollIntoView(str, {}); // $ExpectType Promise<any>
+appium.scrollIntoView(str, true); // $ExpectType Promise<any>
 appium.seeCheckboxIsChecked(str); // $ExpectType Promise<any>
 appium.seeElement(str); // $ExpectType Promise<any>
 appium.seeInField(str, str); // $ExpectType Promise<any>

--- a/typings/tests/helpers/WebDriverIO.types.ts
+++ b/typings/tests/helpers/WebDriverIO.types.ts
@@ -254,6 +254,7 @@ wd.executeAsyncScript(() => {}, {}); // $ExpectType Promise<any>
 
 wd.scrollIntoView(); // $ExpectError
 wd.scrollIntoView('div'); // $ExpectError
+wd.scrollIntoView('div', true);
 wd.scrollIntoView('div', { behavior: 'auto', block: 'center', inline: 'center' });
 
 wd.scrollTo(); // $ExpectError

